### PR TITLE
REACTOR-363: Add resources for flow logs and hide them behind a '--kinesis' flag

### PR DESCRIPTION
**Pull request by imcgaunn**
REACTOR-363

### Description

- [ ] Existing GitHub Issue? Please provide #: 

### Acceptance checklist

- [ ] Feedback addressed
- [ ] Tests added or modified appropriately
- [ ] All tests passed

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -456,5 +457,51 @@ resources:
       Properties:
         RetentionInDays: "3"
     # CloudWatch::logs END
+
+    # Kinesis BEGIN
+    DestinationKinesisStream:
+      Type: "AWS::Kinesis::Stream"
+      Condition: FlowLogsEnabled
+      Properties:
+        Name: "${self:service}-${self:provider.stage}-dest-stream"
```
Comment (silvexis) [inline serverless.yml]: rename dest-stream to reactor-stream

Comment (imcgaunn) [reply serverless.yml]: sure thing!

File: serverless.yml [hunk 2]
```diff
@@ -456,5 +457,51 @@ resources:
       Properties:
         RetentionInDays: "3"
     # CloudWatch::logs END
+
+    # Kinesis BEGIN
+    ReactorKinesisStream:
+      Type: "AWS::Kinesis::Stream"
+      Condition: FlowLogsEnabled
```
Comment (silvexis) [inline serverless.yml]: change to KinesisEnabled

Comment (imcgaunn) [reply serverless.yml]: sure!

**Pull request merged**